packages/circuit-ui/components/List/List.stories.tsx

```diff
@@ -60,23 +61,30 @@ const sizes: ListProps['size'][] = ['one', 'two'];
 
 export const Sizes = (args: ListProps) =>
   sizes.map((size) => (
-    <List
-      key={size}
-      {...args}
-      size={size}
-      noMargin
-      css={spacing({ bottom: 'giga' })}
-    >
-      <ListItems />
-    </List>
+    <>
+      <Body size={size}>
+        Use List size {size} with Body {size} text.
+      </Body>
+      <List
+        key={size}
+        {...args}
+        size={size}
+        noMargin
+        css={spacing({ bottom: 'giga' })}
+      >
+        <ListItems />
+      </List>
+    </>
   ));
 
 export const Nested = (args: ListProps) => (
   <List {...args} noMargin>
-    <ListItems />
-    <List {...args} noMargin>
-      <li>Sometimes a nested list</li>
-    </List>
```

Comment:
This was invalid HTML[^1]:

```html
<ul>
  <li>...</li>
  <ul> <!-- ul shouldn't be a direct child of ul -->
    <li>...</li>
  </ul>
</ul>
```

Instead it should be:

```html
<ul>
  <li>...</li>
  <li> <!-- li can contain a nested ul -->
    <ul>
      <li>...</li>
    </ul>
  </li>
```

[^1]: the ul can contain only li elements ([ul element spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element))